x-pack/plugins/canvas/public/components/workpad_page/event_handlers.js

```diff
@@ -79,8 +95,8 @@ const handleWheel = (
 
 const handleMouseDown = (commit, e, isEditable) => {
   e.stopPropagation();
-  const { target, clientX, clientY, button, altKey, metaKey, shiftKey, ctrlKey } = e;
```

Comment:
Was this a bug? Seems like `button` would always be undefined...

Reply:
The event object has both `button` and `buttons` properties. `button` is the value of which button (i.e. 0 for left click) to trigger the event which works for the `mouseDown` event, but it doesn't work for `mouseMove` event. `buttons` tells you which buttons are being pressed when the event is triggered (i.e. 1 for left click being held down).

Using `button` here isn't actually a bug. I just changed this to `buttons` because it works for both `mouseDown` and `mouseMove` to keep it consistent.

Reply:
This check was put it place to prevent elements from dragging on right click. Changing it to `buttons` still prevents the dragging.